beacon-chain/rpc/service.go

```diff
@@ -499,6 +499,8 @@ func (s *Service) Start() {
 			FinalizationFetcher:   s.cfg.FinalizationFetcher,
 			ChainInfoFetcher:      s.cfg.ChainInfoFetcher,
 		}
+		s.cfg.Router.HandleFunc("/eth/v1/debug/beacon/states/{state_id}", debugServerV1.GetBeaconStateSSZ).Methods(http.MethodGet)
```

Comment:
is v1 only SSZ? i was surprised by this. 
```
case "/eth/v1/debug/beacon/states/{state_id}":
		endpoint.GetResponse = &BeaconStateResponseJson{}
		endpoint.CustomHandlers = []apimiddleware.CustomHandler{handleGetBeaconStateSSZ}
```

seemed like this showed otherwise.

Reply:
I am not sure what happened, but we currently only have the SSZ grpc endpoint. Maybe we wanted to remove it entirely in the previous fork, but accidentally only removed the non-SSZ version? It doesn't matter much because v1 is no longer supported in the spec and we will remove it in the next fork.